Source/WebCore/platform/graphics/FloatSegment.h

```diff
@@ -55,7 +56,7 @@ struct FloatSegment {
 // return differenceWithDilation({ 0, totalWidth }, intersections);
 inline Vector<FloatSegment> differenceWithDilation(FloatSegment a, Vector<FloatSegment>&& bs, float dilationAmount)
```

Comment:
I don't understand what the r-value for `bs` is doing here.

Reply:
It returns it at the end using `WTFMove(bs)`, I guess this was a way to avoid an out-parameter?